GVFS/GVFS.Installer.Mac/scripts/postinstall

```diff
@@ -1,5 +1,26 @@
 #!/bin/bash
 
+function startOrRestartService()
+{
+	domain=$1
```

Comment:
For the LogDaemon case, this is going to be `system` but for the .app, it's going to be `gui/<UID>`, where does the gui come from? Why do we need it? 
I think it's always been like this, but it stands out to me now that there is a separate function.

Reply:
>>where does the gui come from?

`gui/<uid>` is one of the 5 `launchctl` domain specifiers, others include `system`,`user/<uid>`, `login/<audit session id>`, and `session/<audit session id>`.

Domains are like namespaces for services - `launchtl print <domain>` is a great tool to view the services registered in a domain. `launchctl print system` prints all system domain services (they may get launched before user logs in), `launchctl print gui/502` prints all `gui` domain services associated with user id `502` (these get launched when user with uid `502` logs in).

>>Why do we need it?

We need it to tell launchd the domain where our app should get registered for launching. For our GUI App, `gui/<uid>` would be the natural choice. We could also use `login/<asid>` but then we need to provide the `asid`, its a little easier to fetch `uid` and use `gui`.